proposals/README.md

```diff
@@ -0,0 +1,11 @@
+# NIPs(NucliaDB Improvements Proposals)
```

Comment:
Maybe you can do a PR with this file so you don't need to add it in each proposal